components/forms/PasswordReset/index.jsx

```diff
@@ -0,0 +1,38 @@
+export const passwordResetFormTestIds = {
+  form: 'password-reset-form',
+  emailLabel: 'email',
+  emailField: 'input-email',
+  passwordResetButton: 'btn-password-reset',
+};
+
+const PasswordResetForm = () => {
+  return (
+    <form className="w-full" data-test-id={passwordResetFormTestIds.form}>
+      <div className="mb-6">
+        <label
+          className="block text-white text-base-small font-extrabold mb-2"
+          htmlFor="email"
+          data-test-id={passwordResetFormTestIds.emailLabel}
+        >
+          Email
+        </label>
+        <input
+          className="w-full h-14 bg-white/20 text-white text-base-large rounded-xl px-3 py-1.5 focus:outline-none focus:shadow-none"
+          id="email"
+          data-test-id={passwordResetFormTestIds.emailField}
+          type="email"
+          required
+        />
```

Comment:
Should we create an `Input` (or `BaseInput` or `TextInput`) component that contains the styling for all base inputs and takes props like `type` and `name`? :) 

This would DRY the code and enable to update the style easily in the future :)

![image](https://user-images.githubusercontent.com/77609814/160739227-dddd608a-6618-44e5-b9f5-192eb0a53a84.png)

Reply:
In addition to this, you might want to take any props to provide flexibility to override/extend any props that would pass to the `input` e.g.: `className` then append it to the end of the style you added for the base input 💡

Reply:
DRYed out Input and Button components in 7213c8904d5d033f8024dd4dc02223ec1f0b5007
Button component's sizes are chosen depending on the design on figma!